src/local-history.ts

```diff
@@ -138,4 +138,14 @@ export class LocalHistoryProvider {
             this.historyFiles.push(data);
         }
     }
+
+    /**
+     * Compare the two revisions and show the diff between them
+     */
+    private viewDiffBtwRevisions(currentVersion: vscode.Uri, historyVersion: vscode.Uri) {
```

Comment:
[minor] rename `viewDiffBtwRevisions` to `displayDiff(current, previous)`.
[minor] please include a return type (check if we can add a rule for it)